packages/twenty-server/src/core/workspace/services/workspace.service.ts

```diff
@@ -44,6 +49,16 @@ export class WorkspaceService extends TypeOrmQueryService<Workspace> {
 
     assert(workspace, 'Workspace not found');
 
+    await this.userWorkspaceRepository.delete({ workspaceId: id });
+
+    const users = await this.userRepository.find({
+      where: {
+        defaultWorkspaceId: workspace.id,
+      },
+    });
+
+    await this.userRepository.remove(users);
```

Comment:
we don't want to delete users when deleting the workspace, only the workspaceMembers located in the workspaceSchema (already done by `await this.workspaceManagerService.delete(id);`)
```suggestion
```